org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/adwindow/StatusBar.java

```diff
@@ -229,7 +230,15 @@ public void setStatusLine (String text, boolean error, ProcessInfoLog[] m_logs)
 			messageContainer.appendChild(label);
 			label.addEventListener(Events.ON_CLICK, this);
     	}
-    	
+    }
+
+    /**
+     * add document/record link from ProcessInfoLog to popup
+     * @param m_logs
+     * @return
+     */
+    private Div buildPopupContent(ProcessInfoLog[] m_logs) {
```

Comment:
the method name is too similar to the existing createPopupContent, better use "buildProcessLogContent" instead.